pkg/ui/workspaces/db-console/src/views/shared/components/summaryBar/summarybar.styl

```diff
@@ -159,4 +162,4 @@
   &__label
     font-size 14px
     line-height 22px
-    letter-spacing 0.1px
```

Comment:
my GoLand automatically remove the line here, I can revert if it is necessary.

Reply:
Thanks for the explanation! If this isn't relevant/needed for the changes we're making in this PR, let's avoid changing it.